env_setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+ENV_NAME="ev"
+echo $ENV_NAME
+
+# conda update conda
+echo removing environment $ENV_NAME
+conda remove -n $ENV_NAME --all
+
+
+echo 'Create build environment'
+# # See https://github.com/pytorch/vision/issues/7296 for ffmpeg
+conda create \
```

Comment:
Cool, this resolves the concern I had about environment management and package management